packages/flowtype/flowtype.0.40.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+maintainer: "gabe@fb.com"
+homepage: "http://flowtype.org"
+dev-repo: "https://github.com/facebook/flow.git"
+bug-reports: "https://github.com/facebook/flow/issues"
+authors: [
+  "Avik Chaudhuri"
+  "Basil Hosmer"
+  "Gabe Levi"
+  "Jeff Morrison"
+  "Marshall Roch"
+  "Sam Goldman"
+  "James Kyle"
+]
+doc: "http://flowtype.org/docs/getting-started.html"
+license: "BSD3"
+depends: [
+  "base-unix"
+  "base-bytes"
+  "ocamlbuild" {build}
+]
+available: [ocaml-version >= "4.01.0" & ocaml-version < "4.04.0"]
```

Comment:
Looks like the minimum version of OCaml needed is now 4.02.0, due to:
```
# File "src/server/findRefs_js.ml", line 40, characters 7-21:
# Error: Unbound value List.sort_uniq
```

Could this constraint please be bumped to `ocaml-version >= "4.02.0"` ?